src/app/features/events/components/event-card/event-card.component.html

```diff
@@ -88,16 +88,16 @@
                     <div *ngIf="!loading" class="title-larger-text">
                         <div *ngIf="model?.canceled">
                         </div>
-                        <div>
-                            <span class="canceled" *ngIf="model?.canceled">CANCELED </span>{{ model?.title }} <!--[canceled]="translations.eventCard.canceled.canceled_on | translate"-->
+                        <div [attr.canceled]="translations.eventCard.canceled.canceled_on | translate">
+                            <span class="canceled" *ngIf="model?.canceled">CANCELED </span>{{ model?.title }}
                         </div>
                     </div>
 
-                    <div *ngIf="!loading" class="event-organizer"> <!--[organize]="translations.eventCard.organize.organizer | translate"-->
+                    <div [attr.organize]="translations.eventCard.organize.organizer | translate" *ngIf="!loading" class="event-organizer"> 
```

Comment:
Same as my other comment, remove the `[attr.organize]` line and put the translation in the span replacing "Organizer".